var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -94,6 +96,9 @@ def setup_dependent_package(self, module, dependent_spec):
         module.cargo = Executable(os.path.join(self.spec.prefix.bin, "cargo"))
 
     def setup_build_environment(self, env):
+        # libssh2-sys
```

Comment:
```suggestion
        # Manually instruct Cargo dependency ssh2-rs to build with the
        # Spack installed libssh2 package.
```

Reply:
`libssh2-sys` is a crate on its own

Reply:
Ah strange haha, updated in the comment.